addons/website/static/src/js/menu/translate.js

```diff
@@ -178,6 +178,36 @@ var TranslatePageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             devicePreview: false,
         };
 
+        const showTip = ev => {
+            const $target = $(ev.target);
+            let message = _t('This translation is not editable.');
+            if (ev.target.closest('.s_table_of_content_navbar_wrap')) {
+                message = _t('Translate header in the text. Menu is generated automatically.');
```

Comment:
?
We are in generic code not related to table of content
Clicking on any `.o_not_editable [data-oe-translation-id]` will show this tooltip :thinking: ?

Reply:
Yes, it will appear as a yellow block in the screen - and it feels strange if it cannot be modified without explanation.
But I adapted the condition to only consider "non `:o_editable`" elements: (in case there is a chain of on/off editable)
```js
for (const translationEl of $('.o_not_editable [data-oe-translation-id]').not(':o_editable')) {
```